projects/console-web-server/src/db/entity/device.entity.ts

```diff
@@ -71,6 +72,9 @@ export class Device extends BaseEntity implements DeviceBase {
   @Column({ type: 'smallint', name: DevicePropSnake.is_virtual, unsigned: true, default: 0, nullable: false })
   isVirtual!: number;
 
+  @Column({ type: 'character varying', name: DevicePropSnake.display_error, length: DEVICE_DISPLAY_ERROR_MAX_LENGTH, default: '', nullable: false })
+  displayError!: string;
```

Comment:
If `displayError` type is `string | null`, `nullable: false` should be removed.

And this action needs to re-generate migration files